topic_message_query.go

```diff
@@ -219,8 +222,24 @@ func defaultRetryHandler(err error) bool {
 	code := status.Code(err)
 
 	switch code {
-	case codes.NotFound, codes.ResourceExhausted, codes.Internal, codes.Unavailable:
+	case codes.NotFound, codes.ResourceExhausted, codes.Unavailable:
 		return true
+	case codes.Internal:
+		if RST_STREAM == nil {
+			var err1 error
+			RST_STREAM, err1 = regexp.Compile(".*(rst.stream.*internal.error|internal.error.*rst.stream).*")
```

Comment:
Should also add an unit test for at least the retry handler portion of things.

Reply:
Can't really add a unit test since yesterday while testing the Go SDK against testnet node 6 I was getting
Go: 
```
rpc error: code = Unavailable desc = Service Unavailable: HTTP status code 503; transport: received the unexpected content-type "text/html"
```
JS:
```
Error: 13 INTERNAL: Received RST_STREAM with code 0
```
Java:
```
    UNAVAILABLE: HTTP status code 503
    invalid content-type: text/html
    headers: Metadata(:status=503,content-length=107,cache-control=no-cache,content-type=text/html)
    DATA-----------------------------
    <html><body><h1>503 Service Unavailable</h1>
    No server is available to handle this request.
    </body></html>
    DATA-----------------------------
```
In JS I could test a real `RST_STREAM` error I got back from the channel, while in Go I cannot.